openshift/release/generate-release.sh

```diff
@@ -7,12 +7,14 @@ source $(dirname $0)/resolve.sh
 GITHUB_ACTIONS=true $(dirname $0)/../../hack/update-codegen.sh
 git apply openshift/patches/disable-ko-publish-rekt.patch
 git apply openshift/patches/override-min-version.patch
+git apply openshift/patches/add-eventshub-port-name.patch
+git apply openshift/patches/unique-secret-names.patch
```

Comment:
Not sure if this should be here or not? If ok, feel free to unhold.

Reply:
Yes, that's what missing, without this `make generated-files` will revert that patch